components/assets/LogoCcv.tsx

```diff
@@ -1,16 +1,16 @@
 import React from "react"
-import { SvgProps } from "@/components/assets/LogoBrown"
+import { LogoProps } from "@/components/assets/LogoBrown"
```

Comment:
Hmmm.... This is another organization thing, but I personally don't love importing the props from the `tsx` file. I had to solve this in the riddc app, but what I ended up doing was  file structure like this:

```
components
|
| - Icons
  | - index.ts
  | - types.ts
  | - Brown.svg.tsx
  | - /* whatever other logos */
```
And in `types.ts` I defined the shared properties like `LogoProps` though I think I called it something else. 

I don't want to be a stickler or say that this is the only solution, but I'm just really wary around circular imports (which have, in my experience, been caused by me doing things like this).

So! Not a big problem, but I'd suggest moving it around a little? What do you think?